sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/package.scala

```diff
@@ -45,7 +45,7 @@ package object util extends Logging {
     }
   }
 
-  def fileToString(file: File, encoding: String = "UTF-8"): String = {
+  def fileToString(file: File, encoding: String = UTF_8.name()): String = {
```

Comment:
I wonder if we can change these private methods to accept a `Charset`? it'd be tidier.